docs/rules/prefer-number-properties.md

```diff
@@ -93,6 +93,13 @@ Default: `false`
 
 Pass `checkInfinity: true` to enable check on `Infinity`.
 
+### checkNaN
+
+Type: `boolean`\
+Default: `true`
+
+Pass `checkNaN: false` to disable check on `NaN`.
```

Comment:
```suggestion
Default: `false`

Pass `checkNaN: true` to enable check on `NaN`.
```

Reply:
Yes, there are also a number of tests that need to be modified (ran with checkNaN: true), will do them